packages/backend/src/build-disk-image.ts

```diff
@@ -305,8 +305,27 @@ export async function getUnusedName(name: string): Promise<string> {
 }
 
 // Create builder options for the "bootc-image-builder" container
-export function createBuilderImageOptions(name: string, build: BootcBuildInfo): ContainerCreateOptions {
-  const cmd = [`${build.image}:${build.tag}`, '--output', '/output/', '--local'];
+export async function createBuilderImageOptions(name: string, build: BootcBuildInfo): Promise<ContainerCreateOptions> {
+  // check image for builder to use
+  const image = `${build.image}:${build.tag}`;
+
+  const buildProp = await getConfigurationValue<string>('builder');
+
+  // default to centos builder
+  let builder = bootcImageBuilderCentos;
+  if (buildProp === 'rhel') {
+    // switch to rhel if that's the preference
+    builder = bootcImageBuilderRHEL;
+  } else if (buildProp === 'image') {
+    // or switch to rhel if the preference comes from the image label
+    // AND we detect the rhel label
+    const buildLabel = await containerUtils.getImageBuilderLabel(image);
```

Comment:
listImages is very expensive call and I see it's done within getImageBuilderLabel and each time we do it it makes the build "stutter" a bit before building.

i think that we should move this earlier in the call process and use the already done listImages call in the code? so we can avoid calling it twice.

Reply:
Where do we already call listImages()? I agree we should not call twice if we can avoid it but I didn't see an obvious use already.